Jenkinsfile

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env groovy
+
+REPOSITORY = 'local-links-manager'
+
+node {
+   def govuk = load '/var/lib/jenkins/groovy_scripts/govuk_jenkinslib.groovy'
+
+   try {
+      stage("Checkout") {
+         checkout scm
+      }
+
+      stage("Build") {
+         sh "${WORKSPACE}/jenkins.sh"
+      }
+
+      stage("Push release tag") {
+         echo 'Pushing tag'
+         //govuk.pushTag(REPOSITORY, env.BRANCH_NAME, 'release_' + env.BUILD_NUMBER)
```

Comment:
I think it can be uncommented in this repo because it's not enabled here yet: https://github.com/alphagov/govuk-puppet/blob/0575de6cf1f11e4f74ce06b1862eb3e72e7010eb/modules/govuk_jenkins/files/var/lib/jenkins/groovy_scripts/govuk_jenkinslib.groovy#L18